personModel class

dndana opened the pull request:
person model

dndana committed:
personModel class

ivyfangru commented on app/src/main/java/edu/depaul/jmorton/boschhack/boschhackathon/models/PersonModel.java:
```diff
@@ -0,0 +1,67 @@
+    private String firstName;
+    private String lastName;
+    private Date birthdate;
+    private char bloodType;
```
the blood type should be longer than one character like: AB type.

djLarrikin replied:
Yea it should be a String so that something like AB+ is possible too

dndana committed:
personModel class : changed bloodtype to string

Commit:
Merge branch 'development' of https://github.com/djLarrikin/BoschHackathon into feature/DriverModel

dndana committed:
personModel class changes
- bloodtype was changed to string ( AB, AB+)
- gender was changed to char (F,M)

jonathan-morton reviewed (approved):
lgtm
